pkgs/native_assets_cli/lib/src/locking/locking.dart

```diff
@@ -73,6 +80,8 @@ Future<T> _runUnderFileLock<T>(
         );
         printed = true;
       }
+      // Don't busy wait, give the CPU some rest.
+      // Magic constant taken from flutter_tools for startup lock.
```

Comment:
@Hixie I don't see any documentation on https://github.com/flutter/flutter/pull/4476 for how the magic constant was chosen. Would you happen to remember why 50 milliseconds?